_data/risc_outgoing.yml

```
@@ -58,3 +60,55 @@
     }
   }
 
+- friendly_name: Identifier Changed
+  direction: outgoing
+  event_type: https://schemas.openid.net/secevent/risc/event-type/identifier-changed
+  spec_url: "https://openid.net/specs/openid-risc-event-types-1_0-ID1.html#rfc.section.2.5"
+  description: >
+    Login.gov pushes this event when a user changes the email address on their account.
+  payload_schema:
+    *email_schema
+  example_payload: {
+    "https://schemas.openid.net/secevent/risc/event-type/identifier-changed": {
+      "subject": {
+        "subject_type": "email",
+        "email": "<$EMAIL>"
+      }
+    }
+  }
+
+- friendly_name: Recovery Activated
+  event_type: https://schemas.openid.net/secevent/risc/event-type/recovery-activated
+  spec_url: "https://openid.net/specs/openid-risc-event-types-1_0-ID1.html#rfc.section.2.8"
+  description: >
+    Login.gov pushes this event when a user starts the "Forgot Password" flow for an account.
+  payload_schema:
+    *iss_sub_schema
+  example_payload: {
+    "https://schemas.openid.net/secevent/risc/event-type/recovery-activated": {
+      "subject": {
+        "subject_type": "iss-sub",
+        "iss": "https://idp.int.identitysandbox.gov",
+        "sub": "<$SUB>"
+      }
+    }
+  }
+
+- friendly_name: Recovery Information Changed
+  event_type: https://schemas.openid.net/secevent/risc/event-type/recovery-information-changed
+  spec_url: "https://openid.net/specs/openid-risc-event-types-1_0-ID1.html#rfc.section.2.9"
+  description: >
+    Login.gov pushes this event when a user changes any of their second factor, for example,
+    adding or removing an email, phone address, TOTP authentication app, backup codes,
+    or PIV/CAC.
+    payload_schema:
```

Should this have been...

```suggestion
  payload_schema:
```

Currently, it's included in the description text.

---

whoooops, great catch, thank you, PR to fix https://github.com/18F/identity-dev-docs/pull/181